admin/Api/Data/EnvironmentStatusInterface.php

```diff
@@ -1,10 +1,9 @@
 <?php
 
+namespace Pronko\LiqPayAdmin\Api\Data;
 
-namespace Pronko\LiqPayAdmin\Model;
-
-interface ModeStatusInterface
+interface EnvironmentStatusInterface
```

Comment:
Looks 👍 if you can also add PHPDocs here e.g. @api annotation.

Reply:
@mcspronko done )